contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -3601,3 +3601,44 @@ AS SELECT
 FROM sys.objects so
 WHERE FALSE;
 GRANT SELECT ON sys.sequences TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.dm_os_sys_info 
+AS SELECT 
+  CAST(0 AS BIGINT) AS cpu_ticks,
+  CAST(ROUND(EXTRACT(EPOCH FROM NOW()) * 1000.0, 0) AS BIGINT) AS ms_ticks, 
+  CAST(0 AS INT) AS cpu_count,
+  CAST(0 AS INT) AS hyperthread_ratio,
+  CAST(0 AS BIGINT) AS physical_memory_kb,
```

Comment:
physical_memory_in_bytes is still missing

Reply:
the doc says its for **Applies to: SQL Server 2008 (10.0.x) and SQL Server 2008 R2 (10.50.x).** so not added it. I also verified with latest sqlserver 16 and it not present. I think its not required.